packages/clerk-js/src/ui/elements/Card/CardContent.tsx

```diff
@@ -79,6 +82,15 @@ export const CardContent = React.forwardRef<HTMLDivElement, CardContentProps>((p
         <CardAlert variant={'warning'}>{t(localizationKeys('maintenanceMode'))}</CardAlert>
       )}
       {children}
+
+      <DevModeNotice
+        sx={{
+          position: 'absolute',
+          bottom: 0,
+          left: '50%',
```

Comment:
Seems like we can just `left: 0, right: 0, textAlign: 'center'` here instead of the shift left and pull back with transform?